packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -727,6 +727,16 @@ model = pe.VisionTransformer.from_config("${model.id}", pretrained=True)`;
 		return [vision_encoder];
 	}
 };
+export const phantom = (model: ModelData): string[] => [
+	`huggingface-cli download --local-dir ${nameWithoutNamespace(model.id)} --local-dir ./${model.id} # in cli
+from phantom_wan import WANI2V, configs
+wan_i2v = WanI2V(
+            config=configs.WAN_CONFIGS['i2v-14B'],
+            checkpoint_dir=./${model.id}
+        )
+ video = wan_i2v.generate(text_prompt, image_prompt)
+`,
+];
```

Comment:
Agree with @Vaibhavs10 re: using `snapshot_download` directly. Also, no need to download to a local dir IMO (usually better to showcase the shared cache and if user wants to use a local dir, they can always do it themselves)

```suggestion
export const phantom = (model: ModelData): string[] => [
	`from huggingface_hub import snapshot_download
from phantom_wan import WANI2V, configs

checkpoint_dir = snapshot_download("${model.id}")
wan_i2v = WanI2V(
            config=configs.WAN_CONFIGS['i2v-14B'],
            checkpoint_dir=checkpoint_dir,
        )
 video = wan_i2v.generate(text_prompt, image_prompt)`
];
```